### PR TITLE
chore: rename and fix typos from previous version in oauth

### PR DESCRIPTION
**Pull request by Skemba**

### CODE REVIEW

File: pkg/server/oauth_github.go [hunk 1]
```diff
@@ -11,18 +11,18 @@ import (
 	"golang.org/x/oauth2"
 )
 
-type oauthHanlderGithub struct {
+type oauthHandlerGithub struct {
```
Comment (petethepig) [inline pkg/server/oauth_github.go:14]: this is funny, I looked at them so many times and never noticed

File: go.mod [hunk 2]
```diff
@@ -44,6 +44,7 @@ require (
 	github.com/klauspost/compress v1.13.5
 	github.com/kyoh86/richgo v0.3.3
 	github.com/kyoh86/xdg v1.2.0 // indirect
+	github.com/markbates/pkger v0.17.1
```
Comment (petethepig) [inline go.mod:47]: we no longer use pkger, can you run `go mod tidy`?

**Review by petethepig — CHANGES_REQUESTED** [review]
see comments above

File: pkg/server/oauth_github.go (hunk 1 above)
Comment (Skemba) [reply pkg/server/oauth_github.go:14]: me too, it was only once I installed VSCode extension for spelling errors that I noticed

**Pull request merged**